File: chalicelib/ssm.py
import boto3


class SSMClient:
    def __init__(self):
        self._ssm = boto3.client('ssm')

    def get_parameter(self, key: str) -> str:
        """
        Get value from Parameter Store
        """
        response = self._ssm.get_parameter(
            Name=key,
            WithDecryption=True
        )
        return response['Parameter']['Value']

File: src/helper/ssm.go
package helper

import (
	"context"

	"github.com/aws/aws-sdk-go-v2/aws"
	"github.com/aws/aws-sdk-go-v2/service/ssm"
)

func GetParameter(c ssm.Client, name string) (*string, error) {
	o, err := c.GetParameter(context.TODO(), &ssm.GetParameterInput{
		Name:           aws.String(name),
		WithDecryption: true,
	})
	if err != nil {
		return nil, err
	}
	return o.Parameter.Value, nil
}
